pom.xml

```diff
@@ -196,11 +196,17 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>apache-httpcomponents-client-4-api</artifactId>
     </dependency>
+
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
       <version>3.12.0</version>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.11.0</version>
```

Comment:
@jglick do you think that this could go into `bom` ?
I can see from my IntelliJ that `2.6` is defined somewhere in the bom for `2.204` - but when I try to find it in the bom repo, I don't see it anywhere..

Reply:
@basil fyi